### PR TITLE
Prevent tags #6614

### PR DESCRIPTION
**Pull request by rationality6**
### Summary

#6614 issue fix

I need code reviews. There are many mistakes, I'm getting used to Kolibri architecture though.

…

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,8 @@
     "build-kolibri-tools": "yarn workspace kolibri-tools run build-kolibri-tools",
     "publish-packages": "node ./packages/publish.js",
     "hashi-dev": "yarn workspace hashi run dev",
-    "hashi-build": "yarn workspace hashi run build"
+    "hashi-build": "yarn workspace hashi run build",
+    "lint:css": "stylelint '**/*.vue'"
```
Comment (indirectlylit) [inline package.json]: We should not add a new command. Instead, the linting should be performed by the existing command defined above, `lint-frontend`

File: package.json [hunk 2]
```diff
@@ -57,7 +58,9 @@
     "@types/jest": "^24.0.12",
     "black-fmt": "https://github.com/learningequality/black-fmt#v0.1.3",
     "kolibri-tools": "0.13.0-dev.10",
-    "yarn-run-all": "^3.1.1"
+    "yarn-run-all": "^3.1.1",
+    "stylelint": "^13.3.3",
+    "stylelint-selector-tag-no-without-class": "^2.0.3"
```
Comment (indirectlylit) [inline package.json]: `stylelint` is already a dependency of `kolibri-tools` and does not need to be added here.

Additionally, `stylelint` related dependencies should also be added to the `kolibri-tools` package.json file here: 

https://github.com/learningequality/kolibri/blob/4c961110eabf4711da4e2f6c1430c71004e1b6da/packages/kolibri-tools/package.json#L64-L71

**Review by indirectlylit — COMMENTED** [review]
Hey, thanks for your work so far on this! it's looking great and will be a big improvement – nice job.

You'll need to rebase or merge in the latest `develop` to resolve conflicts. Unfortunately this is is inevitable because `develop` is changing rapidly.

I've also flagged below that rather than making new lint commands, we should be integrating with the existing system. Note that `kolibri-tools` is a [Yarn workspace](https://classic.yarnpkg.com/en/docs/workspaces/), so to add new dependencies there you'll need to do something like:

```bash
yarn workspace kolibri-tools add stylelint-selector-tag-no-without-class
```

File: package.json (hunk 1 above)
Comment (rationality6) [reply package.json]: Thank you for reviewing @indirectlylit. I'll do it in that way

File: package.json (hunk 2 above)
Comment (rationality6) [reply package.json]: I've been read the yarn workspace document. I'll take note.

**Comment by rationality6** [comment]
Close for rebase

**Comment by indirectlylit** [comment]
> Close for rebase

Note that you shouldn't need to close the PR in order to rebase or merge

**Comment by rationality6** [comment]
Sorry for my limitation of understanding of Kolibri plugin architecture. I'm reading documentation again. You guys don't need to review these codes. I'll commit it again later. thank you for your commitments.

**Comment by indirectlylit** [comment]
let us know if you have questions. The documentations might be incomplete - happy to help

**Pull request closed without merging**